### ISSUE TITLE
Calendar disables the send and spellcheck buttons

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#190]
```
By enabling the calendar the Send,SpellCheck and Save Draft buttons become 
unusable like if they were disabled but still visible.

Nothing appears at apache or round cube logs

Running on Debian 6 apache2+php5 and the latest roundcube 0.5.2

Also tryed to disable the other plugins but the problem remains. Using Chrome 
or Firefox got the same result


```

Original issue reported on code.google.com by `1c3...@gmail.com` on 15 May 2011 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
Any Javascript errors? Use Firefox and Firebug Addon to check this.
```

Original comment by `rol...@liebl.ath.cx` on 15 May 2011 at 4:51

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in 2.4 bundle. Please cross-check.
```

Original comment by `rol...@liebl.ath.cx` on 15 May 2011 at 6:41
- Changed state: **Invalid**
